### PR TITLE
Phase 2 Trackers T12 and T13: TBPX Lp GBTs placement study

### PR DESCRIPTION
**Pull request by ghugo83**
As discussed with @emiglior and @boudoul, introducing 2 Trackers for a TBPX Lp GBTs placement study. 

* T12: TBPX Lp GBTs are placed around the service cylinder, around TFPX (tkLayout OT614_IT461). Full description at: http://cms-tklayout.web.cern.ch/cms-tklayout/layouts-work/recent-layouts/OT614_200_IT461/materialpixel.html
Geometry scenario is 2023D36.
Workflows are:  232xx, 234xx

* T13: TBPX Lp GBTs are placed around TBPX (tkLayout OT614_IT460). Full description at: http://cms-tklayout.web.cern.ch/cms-tklayout/layouts-work/recent-layouts/OT614_200_IT460/materialpixel.html
Geometry scenario is 2023D37.   
Workflows are: 236xx, 238xx.

NB: 
- The TBPX services are routed as follows, all the way from the barrel modules: 
T12: Directly towards TFPX.
T13: Towards TBPX outermost cylinder first. 
- E-Links, powering, and cooling are all routed together, which is quite an approximation.
- OTST MB update is not integrated here, but does not matter for this study.
- The GBT boards materials descriptions might need more accuracy.
- All overlaps checked.

FYI: @alkemyst

### CODE REVIEW

**Comment by emiglior** [comment]
@skinnari @tomalin this is of interest to you.
This PR updates the T9 and T10 phase2 Tracker geometries (also known as "+20% and +50% inflated passive material") used in the the past for L1-tracking stress tests,
The workflows affected are: 232xx, 234xx, 236xx, 238xx.

**Comment by boudoul** [comment]
Thanks @kpedro88  - Ok let me rephrase to make sure I got the recommendation: we just need to create T12/T13 and D36/D37 resp.,  but we can recycle the previous WF numbers of D26/D27 and reassigned them to D36/D37 , did I get it right ?

**Comment by kpedro88** [comment]
@boudoul yes, if we retire T9/T10 (adding to "deprecated" list in the geometry script), we can reuse the workflow numbers.

**Comment by boudoul** [comment]
ok thanks !  (I should have remembered, but i'm starting to forget all my WF knowledge with time ... :) )   @ghugo83 do you think you can rearrange this PR ? If you need help , please let me know.

**Comment by CHROMIETeledaq** [comment]
Ok cool. Sorry about that, I though the re-use of T9 and T10 had been discussed.
I ll do that then :) I will push the corresponding commits tomorrow (and ping @boudoul if I have an issue).
Thanks to you both :)

**Comment by ghugo83** [comment]
Ok cool. Sorry about that, I though the re-use of T9 and T10 had been discussed.
I ll do that then :) I will push the corresponding commits tomorrow (and ping @boudoul if I have an issue).
Thanks to you both :)

**Comment by kpedro88** [comment]
@ghugo83 thanks! please also update the PR title

**Comment by kpedro88** [comment]
please test workflow 23234.0,23634.0

**Comment by ghugo83** [comment]
Ok so I have done the following:
- Created T12 and T13, and the scenarios D36 and D37.
- Depreciated T9 and T10, and the scenarios D26 and D27.
- D36 and D37 are re-using the workflow numbers of D26 and D27 respectively.

It is the first time I am doing this, have been sneaking around on github, so please let me know for any suggestion :) @boudoul 

Summary: Now we have:
- T12: TBPX Lp GBTs are placed around the service cylinder, around TFPX (tkLayout OT614_IT461). Full description at: http://cms-tklayout.web.cern.ch/cms-tklayout/layouts-work/recent-layouts/OT614_200_IT461/materialpixel.html
Geometry scenario is 2023D36.
Workflows are:  232xx, 234xx

- T13: TBPX Lp GBTs are placed around TBPX (tkLayout OT614_IT460). Full description at: http://cms-tklayout.web.cern.ch/cms-tklayout/layouts-work/recent-layouts/OT614_200_IT460/materialpixel.html
Geometry scenario is 2023D37.   
Workflows are: 236xx, 238xx.

**Comment by cvuosalo** [comment]
+1

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+operations

the update of master Configurations is coherent with the purpose of the PR

**Comment by fabiocos** [comment]
@prebello @zhenhu could you please check and sign?

**Comment by fabiocos** [comment]
@ghugo83 could you please update the title of this PR according to the changes implemented?

**Comment by zhenhu** [comment]
+1

**Comment by fabiocos** [comment]
+1
